script/Arcueid/Day 8/08_00_ARC08_2.txt

```diff
@@ -243,12 +248,12 @@ It seems like she thoroughly believes that Arcueid and I are going out.
 [sha:c938b9043864492e60dce047568a995ab27d47ee]{
 -- Page 74, Offset 19856.
 -- 「あのな、秋葉。勘違いもそこまでにしてくれ。
-"Hey Akiha. Could you stop with the misunderstandings?
+"Seriously, Akiha, you're barking up the wrong tree.
```

Comment:
I don't think this is the right phrase to use here.
Maybe something like

>Seriously, Akiha; could you not jump to all these hasty conclusions?

or something like that is more appropriate.

Reply:
Hmm, I do think it works - she has (allegedly) misread the situation. Suggestion is a little clunky for natural speech. You could do like
> Seriously, Akiha, could you quit jumping to conclusions [already]?

but I still don't love it, since I want it to convey more "you've got the wrong idea" than "your thought process is hasty".

Reply:
I'm not sure exactly what if any of those are issues. At least not to me. As I've said with idioms, they come with a price, and in this case it feels liable to be misinterpreted.